src/olympia/users/models.py

```diff
@@ -208,8 +208,7 @@ def is_staff(self):
         the django admin tools. Does not guarantee that the user will then
         be able to do anything, as each module can have its own permission
         checks. (see has_module_perms() and has_perm())"""
```

Comment:
We should reword that slightly. I suggest (doesn't let me do that as a github suggestion):
```
        """Property indicating whether the user is a Mozilla Corp employee or
        not. Django uses it to allow access to admin tools, but it does not
        guarantee that they would then be able to do anything as each module
        can have its own permission checks. (see has_module_perms() and
        has_perm()). On top of this, our admin tools should only be
        available over VPN for users part of a specific VPN group."""

(Pretty verbose, feel free to adjust)

Can we also double-check with @wagnerand that it's ok to restrict the admin to users with a `@mozilla.com` email ?

Reply:
ha well, that broke lots of tests :(